devops/dependencies.json

```diff
@@ -74,4 +74,4 @@
       "root": "{ARCHIVE_ROOT}/comp/oclfpga/win"
     }
   }
-}
\ No newline at end of file
+}
```

Comment:
The line was automatically added by Github editor, if it shouldn't be modified, please let me know.